website/content/en/highlights/2023-01-17-0-27-0-upgrade-guide.md

```diff
@@ -0,0 +1,29 @@
+---
+date: "2023-01-17"
+title: "0.27 Upgrade Guide"
+description: "An upgrade guide that addresses breaking changes in 0.27.0"
+authors: ["spencergilbert"]
+release: "0.27.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+Vector's 0.27.0 release includes **breaking changes**:
```

Comment:
I think we should separate this out into a new section since it isn't breaking in the sense that we usually use the term in that existing configurations will continue to run. Maybe "includes **potentially impactful changes**"?

Reply:
https://github.com/vectordotdev/vector/pull/15559/commits/ae836ac8233f44e02004ee1fd408a42d733a005c